### PR TITLE
Update Sublime Text 3 to the latest stable build 3170

### PR DESCRIPTION
**Pull request by jamesl33**
Update Sublime Text 3 to version [3170](https://www.sublimetext.com/blog/articles/sublime-text-3-point-1).

Updating:
* Sublime Text to version 3170

Tested:
Build, install and execution.

### CODE REVIEW

File: programming/sublime-text-3/pspec.xml [hunk 1]
```diff
@@ -31,6 +31,14 @@
...
 
     <History>
+        <Update release="5">
+            <Date>07-05-18</Date>
+            <Version>3170</Version>
+            <Comment>Update to 3170</Comment>
+            <Name>James Lee</Name>
+            <Email>jamesl33info@gmail.com</Email>
+        </Update>
+
```
Comment (DataDrake) [inline programming/sublime-text-3/pspec.xml]: Can you drop these extra whitespace lines? Thanks.

**Comment by jamesl33** [comment]
I've removed those extra newlines

**Comment by selwynorren** [comment]
Hi,

Any idea when this will be approved?

**Comment by jamesl33** [comment]
I don't know but in the meantime you could just download it directly from my fork. That is how I am running it currently.
`sudo eopkg bi --ignore-safety https://raw.githubusercontent.com/jamesl33/3rd-party/UpdateSublime3170/programming/sublime-text-3/pspec.xml`
`sudo eopkg it sublime*.eopkg;sudo rm sublime*.eopkg`

**Comment by selwynorren** [comment]
Hey James,
awesome thanks so much. I appreciate that

**Comment by DataDrake** [comment]
LGTM. Thanks!

**Pull request merged**